ports/sqlite3/CMakeLists.txt

```
@@ -45,6 +46,13 @@ if (NOT WIN32)
     endif()
 endif()
 
+if(SQLITE_ENABLE_ICU)
+    find_package(ICU COMPONENTS uc i18n REQUIRED)
+    target_link_libraries(sqlite3 PRIVATE ICU::uc ICU::i18n)
+
+    string(APPEND PKGCONFIG_REQUIRES_PRIVATE "icu-uc icu-i18n")
```

Nit-pick: When appending to such a string, it is good practice to include a leading ` `, to be independent of order of appending.
```suggestion
    string(APPEND PKGCONFIG_REQUIRES_PRIVATE " icu-uc icu-i18n")
```
No change requested now, it is the single modifying access.

---

Yup, makes sense. 

I will fix this up after this MR gets merged because CI takes a while and I kinda need these changes. 😅